.github/workflows/release_notifications.yaml

```diff
@@ -0,0 +1,18 @@
+name: Send mail on release
```

Comment:
Should we change the name, as it doesn't actually send out mails?

Reply:
Done.